stable/appmesh-controller/ci/values.yaml

```diff
@@ -1,7 +1,7 @@
-# CI testing values for appmesh-controller.
+# CI testing values for appmesh-controller
 
 region: us-west-2
 image:
-  repository: stefanprodan/aws-app-mesh-controller-for-k8s
-  tag: v0.2.0
+  repository: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller
+  tag: v1.0.0-rc5
```

Comment:
Shouldn't the default be the current stable version (0.2.0) with an option to opt-in to the release candidate?

Reply:
Also, image repository should be regionalized.

Reply:
The charts will be not published by this check-in so current users will continue to get v0.2.0. Charts will be published when we build a new tag at the release time. Keeping v0.2.0 here will not help as all the configuration changes/crds are not compatible with v0.2.0 image

Reply:
> Also, image repository should be regionalized.

That's a good idea to make it configurable. I will mark this for a follow up change